src/upstart/radosgw.conf

```diff
@@ -4,7 +4,7 @@ start on radosgw
 stop on runlevel [!2345] or stopping radosgw-all
 
 respawn
-respawn limit 5 30
+respawn limit 3 1800
```

Comment:
Hmm, do we want to change RGW? Unlike the other daemons its state is fairly transient.